projects/core/components/hosted-dropdown/test/hosted-dropdown.component.spec.ts

```diff
@@ -232,11 +230,7 @@ describe(`TuiHostedDropdown`, () => {
         });
 
         it(`Clicking on the host does not open the dropdown`, () => {
-            const element = testComponent.tuiTextfield.nativeFocusableElement;
-
-            tuiAssertIsElement(element);
-
-            element.click();
+            testComponent.tuiTextfield.nativeFocusableElement!.click();
```

Comment:
`testComponent.tuiTextfield.nativeFocusableElement?.click();`

Reply:
This doesn't work for functions that don't return values.